media/css/firefox/whatsnew/whatsnew-109-eu.scss

```diff
@@ -49,9 +49,5 @@
         .wnp-main-img-dark {
             display: block;
         }
-
-        .wnp-main-tagline {
```

Comment:
**Nit (blocking):** By removing this rule, the tagline in darkmode for EU pages is black, making it unreadable:

<img width="703" alt="Screen Shot 2023-01-12 at 5 43 11 PM" src="https://user-images.githubusercontent.com/42309026/212197017-c6677ccd-90c6-40a6-a802-1787110d2a47.png">

Reply:
I don't know why it's removed here, the extra commit came in a rebase and I couldn't get rid of it. I'll fix it.